src/components/List/index.js

```diff
@@ -20,20 +20,24 @@ export default function List (props) {
   typeConvert[requestString] = 'requests';
 
   const [category, setCategory] = useState('All');
-  const [data, setData] = useState([]);
+  const [data, setData] = useState(null);
   useEffect(() => {
     api.getData()
       .then(data => setData(data));
   }, [])
 
+  // before data has returned - any condition other than an empty array
+  if (!data) {
+    return <Message
+      spinner={true}
```

Comment:
You do not need to explicitly say `true` with a boolean literal. Some eslint rules do not allow you to do this at all.
```suggestion
      spinner
```